src/core/profiles/w3c/ldp.rs

```diff
@@ -9,69 +9,37 @@ use crate::profiles::{
     CredentialRequestProfile, CredentialResponseProfile,
 };
 
-use super::{CredentialDefinition, CredentialOfferDefinition};
+use super::{CredentialDefinitionLD, CredentialOfferDefinitionLD};
 
 #[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
 pub struct Metadata {
     credential_signing_alg_values_supported: Option<Vec<String>>,
-    #[serde(rename = "@context")]
-    context: Vec<serde_json::Value>,
```

Comment:
Are you sure about this? Looking at the examples you still have a context for the credential configuration, and a context for the credential definition

Reply:
Actually, follow-up to that -- `Metadata` would benefit from being renamed `CredentialConfiguration` after the draft 13 changes

Reply:
I'll read these sections again to make sure.

Reply:
Yeah, your point was correct, I reverted the change.